src/logic.service.ts

```diff
@@ -219,6 +239,11 @@ export const handlePacs002 = async (transaction: Pacs002): Promise<any> => {
     throw err;
   }
 
+  transaction.DataCache = {
+    "cdtrAcctId": transactionRelationship.from,
```

Comment:
Here you set cdtrAcctId to the accounts/hash , instead of the actual id or hash.

Reply:
This is now retrieved from Redis/Database